cro/cro.py

```diff
@@ -131,44 +133,74 @@ def broadcastspawning(self, REEF, REEFpob):
         ESlarvae = np.concatenate([ESlarvae1, ESlarvae2])
         return ESlarvae
 
-    def brooding(self, REEF, REEFpob, type_brooding='op_mutation'):
+    
+    def _larvaemutation(self, larvae, pos, delta=1):
+        """
+        Description:
+            This function modifies the larvae with mutation 
+        Input:
+            - larvae: new individuals to be mutated
+            - pos: selected positions to be mutated
+            - delta: represents an increment or decrement in each mutation (it could be placed as arg in param_grid)
+        Output:
+            - larvae: modified individuals
+        """
+        (nlarvaes, b) = larvae.shape
+        MM = np.zeros([nlarvaes, b], int) # Mutation matrix
+        # int just for disc mode, replace when cont mode takes place
+
+        for key, value in self.param_grid.items():
+            m = value[0]
+            M = value[1]        
+
+        inc = (M - larvae[range(nlarvaes), pos])
```

Comment:
Why not this?
```python
inc = (M - larvae[:, pos])
```
Maybe I'm missing something...

Reply:
For sure it isn't the same, with range function it iterates automatically in rows. Anyway, I am gonna review it: 

```python
larvae = array([[1, 3, 1],  [0, 9, 1]])
pos = [0, 2]
```

```python
larvae[range(nlarvae), pos]
Out: array([1, 1])
```

```python
larvae[:, pos]
Out: array([[1, 1], [0, 1]])

```